cockroachcloud/migrations-page.md

```diff
@@ -119,24 +186,26 @@ To remove or add a statement, click the ellipsis above the statement and then cl
+| Unimplemented feature                            | The feature does not yet exist on CockroachDB. Implement a workaround by editing the statement and adding statements. Otherwise, remove the statement from the schema. If a link to a tracking issue is included, click the link for further context. For more information about unimplemented features, see [Migrate Your Database to CockroachDB](../{{version_prefix}}migration-overview.html#unimplemented-features). |
+| Statement error                                  | Edit the statement to fix the error. Otherwise, remove the statement from the schema.                                                                                                                                                                                                                                                                                                                                     |
+| Not executed                                     | Remove the statement from the schema. You can include it when [moving data to the new database](../{{version_prefix}}migration-overview.html#step-2-move-your-data-to-cockroachdb).                                                                                                                                                                                                                                       |
+| Missing user                                     | Click the **Add User** button next to the error message. You must be a member of the [`admin` role](user-authorization.html). This adds the missing user to the cluster.                                                                                                                                                                                                                                                  |
+| Incidental error                                 | Resolve the error in the earlier failed statement that caused the incidental error.                                                                                                                                                                                                                                                                                                                                                    |
+| Incompatible statement (non-PostgreSQL dialects) | There is no equivalent syntax on CockroachDB. Implement a workaround by replacing the statement. Otherwise, remove the statement from the schema. Then check **Acknowledge**.                                                                                                                                                                            |
```

Comment:
maybe worth noting statements do not execute until all incompat statements are executed

Reply:
I tried adding a third column. Maybe this makes the table clearer?